.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```

Comment:
Is `eol=lf` actually needed ?
I think developers using Windows should be able to choose favorite eol because Git will convert crlf to lf with `text=auto`.

Tell me about a problem if exists. I don't have windows machine.

Reply:
I'm sadly working on a Windows 10 PC and to be honest, Git for Windows is a pain for multi-env developpement.
I did setup Git to convert CRLF into LF but it goes back to CRLF everytime i'm checking to another branche etc...

VS Code, Sublime and WebStorm work perfectly with LF on Windows, that's why I suggested to force LF directly with this gitattributes file.

Reply:
Thank you, I'm merging this.
Some environments might need to be re-cloned repository.